docs/advanced-features/tabulator.md

```diff
@@ -106,11 +108,27 @@ order to query those tables.  This can be done by users via the per-bucket
 5. **Athena VPC**: If you are using a VPC endpoint for Athena, you must ensure
    it is accessible from the Quilt stack and Tabulator lambda.
 
+### Unrestricted Access
+
+> Available since Quilt Platform version 1.57
+
+By default, Tabulator is only accessible via a session provided by the Quilt Catalog,
+and the access is scoped to the permissions of the Catalog user associated with
+that session. However, an admin can enable **unrestricted access** to Tabulator,
+deferring all access control to AWS. The underlying data in S3 is accessed using
```

Comment:
maybe we should add some docs on how to grant access to tabulator in this case

Reply:
yeah i guess that would be helpful. is it just the tabulator data catalog and everything inside it?

Reply:
I think it's lambda and data catalog